GhostProjectBackend/settings.py

```diff
@@ -74,11 +88,36 @@
 # Database
 # https://docs.djangoproject.com/en/4.2/ref/settings/#databases
 
+# DATABASES = {
+#     'default': {
+#         'ENGINE': 'django.db.backends.sqlite3',
+#         'NAME': BASE_DIR / 'db.sqlite3',
+#     }
+# }
+
 DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
-    }
+  'default': {
```

Comment:
Also, database credentials should be loaded from the env file. This env should not be committed and pushed to the git repository, instead, you can create an additional one for example env.example which will contain all required keys with random values. 
NAME=your_database_name
USER=database_user
etc.
You can refer to the following link https://djangocentral.com/environment-variables-in-django/.